api/src/pcapi/sandboxes/scripts/creators/industrial/utils.py

```diff
@@ -23,3 +29,11 @@ def get_price_by_short_name(occurrence_short_name: str | None = None) -> Decimal
         return Decimal(0)
 
     return Decimal(str(sum(map(ord, occurrence_short_name)) % 50))
+
+
+def create_products_thumb(products: list[offers_models.Product]) -> None:
+    image_dir = pathlib.Path(pcapi.__path__[0]) / "sandboxes" / "thumbs" / "generic_pictures"
```

Comment:
il me semble qu'on n'est pas obligé d'écrire le chemin avec des chaînes de caractères magiques
```suggestion
    from pcapi.sandboxes.thumbs import generic_pictures
    image_dir = pathlib.Path(generic_pictures.__path__[0])
```

Reply:
Yep ! ça marche bien :+1: